docs/application/web/tutorials/app-filtering.md

```diff
@@ -333,6 +335,11 @@ Tizen 4.0.
 
 | Feature key                              | Description                              | Since |
 | ---------------------------------------- | ---------------------------------------- | ----- |
+| `http://tizen.org/feature/network.bluetooth` | Specify this key, if the application requires the Bluetooth feature. | 6.0 |
+| `http://tizen.org/feature/network.bluetooth.audio.media` | Specify this key, if the application requires the Bluetooth Advanced Audio feature (A2DP) feature for using the [Media Key](../api/latest/device_api/tv/tizen/mediakey.html) API. | 6.0   |
```

Comment:
Thanks for the update.

However, I have questions related with this added features.

**Reference :** https://docs1.stg.tizen.org/staging/1233/application/web/api/6.0/device_api/tv/tizen/bluetooth.html#api-features

- According to the Web tv api page, `http://tizen.org/feature/network.bluetooth.audio.media` and `http://tizen.org/feature/network.bluetooth.health` are not related with tv.

- Also this link to Media Key is broken as that page doesn't exist under tv.

- In API pages of mobile/wearable/tv, `http://tizen.org/feature/network.bluetooth.audio.media` is not listed in "Related Feature" sction.

Would you please check these doubts?

Reply:
Thanks a lot for your review.

- As mediakey is not supported on TV, I removed http://tizen.org/feature/network.bluetooth.audio.media. 
http://tizen.org/feature/network.bluetooth.health - this feature is not supported on TV so I also removed it.

- Fixed by removing this reference

- network.bluetooth.audio.media feature is related to mediakey module, so it is not listed in bluetooth module. I unified description for mobile profile to have exact the same information as wearable.

If all comments are resolved please merge using 'squash and merge' policy.

Reply:
All my doubts are resolved. Thanks for the checking.

Hello, @lbardeli. Would you please check the updated files and confirm them again to merge this PR?